src/core/componentSystems/PanelManagementSystem.js

```diff
@@ -0,0 +1,83 @@
+import { MRSystem } from 'mrjs/core/MRSystem';
+import { MRPanel } from '../entities/MRPanel';
+
+
+/**
+ * @class PanelManagementSystem
+ * @classdesc A system that manages the screen relative position of UI panels
+ * @augments MRSystem
+ */
+export class PanelManagementSystem extends MRSystem {
```

Comment:
`PanelManagementSystem` titling is misleading since we also have object managers as well as systems - renaming to something like `MultiPanelSystem` or `PanelSystem` makes more sense

Reply:
yeah I was trying to tie it to Window Management but you're probably right.